src/r2mm/launching/runners/linux/SteamGameRunner_Linux.ts

```diff
@@ -80,7 +83,7 @@ export default class SteamGameRunner_Linux extends GameRunnerProvider {
 
     }
 
-    private async ensureWineWillLoadBepInEx(game: Game): Promise<void | R2Error>{
+    private async ensureWineWillLoadBepInEx(game: Game, proxyDll: string): Promise<void | R2Error>{
```

Comment:
After the additional parameter, the method name is now a bit misnomer. Would `ensureWineWillLoadDoorstop` be more suitable?

Also I've no way to test this works on Linux but I assume you've done so?

Reply:
Doorstop is a Unity-specific thing, so I'm going with "ensureWineWillLoadDllOverride".